specs/deneb/validator.md

```diff
@@ -44,7 +44,8 @@ Note: This API is *unstable*. `get_blobs_and_kzg_commitments` and `get_payload`
 Implementers may also retrieve blobs individually per transaction.
 
 ```python
-def get_blobs_and_kzg_commitments(payload_id: PayloadId) -> Tuple[Sequence[BLSFieldElement], Sequence[KZGCommitment]]:
+def get_blobs_and_kzg_commitments(payload_id: PayloadId) -> \
+        Tuple[Sequence[BLSFieldElement], Sequence[KZGCommitment], Sequence[KZGProof]]:
```

Comment:
Hmm, this is a weird one on style. Generally we try not to have the `\` usage in the specs although I'm unsure at first glance how to do this in our usual style

@hwwhww do you have any ideas?

Reply:
It works with:
```python
def get_blobs_and_kzg_commitments(
    payload_id: PayloadId
) -> Tuple[Sequence[BLSFieldElement], Sequence[KZGCommitment], Sequence[KZGProof]]:
```

Reply:
Fair enough, I will use this. I disagree that this is better though which is why I'm skeptical of linters :P